spec/unit/puppet/provider/network_vlan/cisco_ios_spec.rb

```diff
@@ -18,5 +30,30 @@ def self.load_test_data
     end
   end
 
-  it_behaves_like 'a noop canonicalizer'
+  context 'Vlan defaults throw error tests:' do
+    ['1', '1002', '1003', '1004', '1005'].each do |vlan_name|
+      ['absent', 'create', 'present'].each do |action|
+        action_name = if action == 'create'
+                        'created'
+                      elsif action == 'absent'
+                        'deleted'
+                      else
+                        'updated'
+                      end
+        it "Vlan number #{vlan_name} throws warning and returns false" do
+          expect { described_class.validate_vlan(vlan_name, action) }.to raise_error(RuntimeError, "VLAN #{vlan_name} is a Cisco default VLAN and may not be #{action_name}.")
+        end
+      end
+    end
+  end
+
+  context 'Vlan non-defaults do not throw error tests:' do
+    ['2', '1001', '1006'].each do |vlan_name|
+      ['absent', 'create', 'present'].each do |_action|
```

Comment:
There is a loop here but inside the expect it is always `absent`?

Reply:
removed the loop. seems to be a copy/paste remnant from the test above